packages/libs/core/src/images/imageOptimizer.ts

```diff
@@ -115,6 +116,20 @@ export async function imageOptimizer(
 
   let isAbsolute: boolean;
 
+  /**
+   * Basepath, needs to be removed from the query
+   * 
+   * Not normalised -> error 403
+   * /<base-path>/_next/image?url=%2F<base-path>%2Fassets%2Fimages%2Flogo.svg&w=256&q=75
+   * 
+   * Normalised -> 200
+   * /<base-path>/_next/image?url=%2Fassets%2Fimages%2Flogo.svg&w=256&q=75
+   * 
+   * 
+   */
+  if (url.startsWith(basePath)) {
+    url = url.slice(basePath.length);
+  }
```

Comment:
Maybe add blank line after this condition for readability

Reply:
Done! 😛